docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -233,408 +122,121 @@ is used.
+  :widths: 40, 60
+  :header-rows: 1
 
-The ``COMMENT`` option is supported for adding table columns
-through the :doc:`/sql/alter-table` operations.
+  * - Property name
+    - Description
+  * - ``iceberg.rest-catalog.uri``
+    - REST server API endpoint URI (required). 
+      Example: ``http://iceberg-with-rest:8181``
+  * - ``iceberg.rest-catalog.warehouse``
+    - Warehouse identifier/location for the catalog (optional).
+      Example: ``s3://my_bucket/warehouse_location``
+  * - ``iceberg.rest-catalog.security``
+    - The type of security to use (default: ``NONE``).  ``OAUTH2`` requires
+      either a ``token`` or ``credential``. Example: ``OAUTH2``
+  * - ``iceberg.rest-catalog.session``
+    - Session information included when communicating with the REST Catalog.
+      Options are ``NONE`` or ``USER`` (default: ``NONE``).
+  * - ``iceberg.rest-catalog.oauth2.token``
+    - The bearer token used for interactions with the server. A
+      ``token`` or ``credential`` is required for ``OAUTH2`` security.
+      Example: ``AbCdEf123456``
+  * - ``iceberg.rest-catalog.oauth2.credential``
+    - The credential to exchange for a token in the OAuth2 client credentials
+      flow with the server. A ``token`` or ``credential`` is required for
+      ``OAUTH2`` security. Example: ``AbCdEf123456``
 
-The connector supports the command :doc:`COMMENT </sql/comment>` for setting
-comments on existing entities.
+.. code-block:: text
 
-.. _iceberg-data-management:
+    connector.name=iceberg
+    iceberg.catalog.type=rest
+    iceberg.rest-catalog.uri=http://iceberg-with-rest:8181
 
-Data management
-^^^^^^^^^^^^^^^
+REST catalog does not support :doc:`views</sql/create-view>` or
+:doc:`materialized views</sql/create-materialized-view>`.
 
-The :ref:`sql-data-management` functionality includes support for ``INSERT``,
-``UPDATE``, ``DELETE``, and ``MERGE`` statements.
+.. _iceberg-jdbc-catalog:
 
-.. _iceberg-delete:
+JDBC catalog
+^^^^^^^^^^^^
 
-Deletion by partition
-~~~~~~~~~~~~~~~~~~~~~
+.. warning::
 
-For partitioned tables, the Iceberg connector supports the deletion of entire
-partitions if the ``WHERE`` clause specifies filters only on the identity-transformed
-partitioning columns, that can match entire partitions. Given the table definition
-from :ref:`Partitioned Tables <iceberg-tables>` section,
-the following SQL statement deletes all partitions for which ``country`` is ``US``::
+  The JDBC catalog may face the compatibility issue if Iceberg introduces breaking changes in the future.
+  Consider the :ref:`REST catalog <iceberg-rest-catalog>` as an alternative solution.
 
-    DELETE FROM example.testdb.customer_orders
-    WHERE country = 'US'
+At a minimum, ``iceberg.jdbc-catalog.driver-class``, ``iceberg.jdbc-catalog.connection-url`` and
+``iceberg.jdbc-catalog.catalog-name`` must be configured.
+When using any database besides PostgreSQL, a JDBC driver jar file must be placed in the plugin directory.
```

Comment:
JAR file

Reply:
and trino-iceberg plugin directory

Reply:
This is existing content. This PR won't change existing content, only reorder it as per agreement with maintainers.